mapeditor/jsonutils.h

```diff
@@ -12,6 +12,8 @@
 #include <QVariant>
 #include "../lib/JsonNode.h"
 
+VCMI_LIB_NAMESPACE_BEGIN
```

Comment:
Why should it be part of lib namespace?

Reply:
JsonUtils namespace is already part of vcmi library (lib/JsonUtils,h).
Including any of its extensions into VCMI_LIB namespace is easier. 
Otherwise every usage of JsonUtils namespace in editor will have to specify which namespace to access - global ::JsonUtils or VCMI_LIB::JsonUtils from library.

Another option would be to rename editor version of this namespace to something else, e.g. JsonEditor/JsonQt/etc.